Classes/Private/Manager/Audio/OCTAudioEngine.h

```diff
@@ -35,12 +38,14 @@
 - (BOOL)stopAudioFlow:(NSError **)error;
 
 /**
- * Send the audio to the speaker
- * @param speaker YES to send audio to speaker, NO to reset to default.
- * @param error Pointer to error object.
- * @return YES if successful override, otherwise NO.
+ * Set the input device (not available on Mac OS X).
```

Comment:
Looking at the code even more, I would create separate API for OS X and iOS. IMO no need for similar interface here, this way it would be less confusing.

```objc
#if TARGET_OS_IPHONE

- (BOOL)routeAudioToSpeaker:(BOOL)speaker error:(NSError **)error;

#else

- (BOOL)setInputDeviceID:(NSString *)inputDeviceID error:(NSError **)error;
- (BOOL)setOutputDeviceID:(NSString *)outputDeviceID error:(NSError **)error;

#endif
```